pkg/cloudprovider/providers/aws/aws_loadbalancer_test.go

```diff
@@ -222,3 +224,121 @@ func TestSecurityGroupFiltering(t *testing.T) {
 	}
 
 }
+
+func TestSyncElbListeners(t *testing.T) {
+	tests := []struct {
```

Comment:
I would add a testcase name so people have an idea what you are trying to test at a glance.

Reply:
ah, beat me to it ;)

Reply:
done